package.json

```diff
@@ -39,6 +39,7 @@
     "inquirer": "^7.1.0",
     "keypather": "^3.0.0",
     "marked": "^0.8.2",
+    "moment": "^2.28.0",
```

Comment:
Not a blocker at all, just for future reference, I think it's a good idea to introduce new dependencies/packages in their own commit, explaining what they'll be used for. 

Obviously this is one big commit and it can be time consuming to do atomic commits with our tight deadlines (and this isn't a repo where everything should be perfect), so don't worry about it here.

Reply:
Noted, thanks 👍